src/imgui/imgui_raii_wrapper.h

```diff
@@ -0,0 +1,162 @@
+#ifndef MYPROJECT_IMGUI_RAII_WRAPPER_H
+#define MYPROJECT_IMGUI_RAII_WRAPPER_H
+
+#include <optional>
+#include <string>
+#include <vector>
+
+#include <imgui.h>
+
+namespace irw {
+
+class Windowzita {
+public:
```

Comment:
What's the difference between public and private? When to use one or the other?

Reply:
https://en.cppreference.com/w/cpp/language/access

Reply:
Good question. `public`, `protected` and `private` are access modifiers. These are present in one form of another in a lot of other languages. Python doesn't have it, but they usually use the famous `dunder` and such to mimic access modifiers. Learning this well is very important.